src/components/logos/CDSS.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+import cdssLogo from '../../assets/svg/cdssLogo.svg';
+
+const CDSS = styled.img.attrs({
+  alt: 'cdss.ca.gov',
+  src: cdssLogo
+})`
+  height: 34px;
+  width: 136px;
```

Comment:
Setting thee height and width doesn't seem necessary as it's hardcoded in the svg. Also, if you switch to png, you'll get these exact dimensions.